AirSkyBoat/AirSkyBoat
# Gustaberg NM Audit

**_I affirm:_**

## What does this pull request do?

Audited all NMs in the Gustaberg region for current retail behavior. Removed random window for lottery pop NMs to have a minimum respawn instead of a range. 

All data is taken from a mixture of FFXIclopedia and FFXIDB.

## Steps to test these changes

See changes to NMs in change list and test any logic adjustments, for example Leaping Lizzy is no longer allowed to have two up at the same time.
